python/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
```

Comment:
This pyproject is only for the docsite?

Reply:
Looks like it, we'd also get the benefit that if the top-level project can't resolve dependencies, we know there's a compatibility issue between libraries

Reply:
Yes, this will be for the docsite, you can also use poetry install on this and avoid the need to change your python interpreter on vscode everything you work on something different :)